src/main/resources/CU-ApplicationResources.properties

```diff
@@ -159,6 +159,8 @@ error.iWant.document.completeOption.required=Please specify whether the transact
 error.iWant.document.requisition.exists=A related requisition already exists for this document.
 error.iWant.document.creator.cannot.route.to.self=You cannot enter your own Net ID as the Approver. Only enter a Net ID if you are required to Submit to an approver before this transaction goes to the BSC. Hit Submit when you are ready to route the I Want Doc for further processing.
 error.iWant.document.approver.cannot.route.to.self=As the Approver, you cannot route this document to yourself. Please remove your Net ID and hit the Approve button if you are ready to approve this transaction.
+message.iwant.document.approve.finalized=Approver {0} has indicated the I Want Doc should be finalized and not route to the Service Center.
+message.iwant.document.approve.submit.to.workflow=Approver {0} has sent this I Want Doc forward for further approvals.
```

Comment:
In the submit-to-workflow message, you might want to mention that the doc is specifically routing forward to the service center, for consistency with the radio option text in the "Order Completed Information" section.

Reply:
On the main user story, Cathy said she wants the submit-to-workflow text to be intentionally vague about who the next approval recipient is. Given that, I'm fine with how this submit-to-workflow message is worded, but I'm wondering if the finalized-doc message above it should be updated to be vague about what routing was skipped.